components/dashboards-web-component/package-lock.json

```diff
@@ -2429,7 +2429,7 @@
     "d3": {
       "version": "4.13.0",
       "resolved": "https://registry.npmjs.org/d3/-/d3-4.13.0.tgz",
-      "integrity": "sha512-l8c4+0SldjVKLaE2WG++EQlqD7mh/dmQjvi2L2lKPadAVC+TbJC4ci7Uk9bRi+To0+ansgsS0iWfPjD7DBy+FQ==",
```

Comment:
Why `package-lock.json` is changes, without any changes in dependencies

Reply:
This is caused due to rimraf module execution when running a maven clean will fix the issue in another PR

Reply:
OK